modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala

```diff
@@ -211,11 +229,24 @@ trait VoTableParser {
       }
     }
 
-    def parseSiderealTarget: ValidatedNec[CatalogProblem, SiderealTarget] =
-      (parseName, parseSiderealTracking, parseMagnitudes, parseAngularSize).mapN {
-        (name, pm, mags, angSize) =>
-          SiderealTarget(name, pm, SortedMap(mags.fproductLeft(_.band): _*), angSize)
-      }
+    def parseSiderealTarget: ValidatedNec[CatalogProblem, Target.Sidereal] =
+      (parseName, parseSiderealTracking, parseBandBrightnesses, parseCatalogInfo, parseAngularSize)
+        .mapN { (name, pm, mags, info, angSize) =>
+          Target.Sidereal(
+            name,
+            pm,
+            // We set arbitrary values for `sourceProfile`, `spectralDefinition`, `sed` and  `librarySpectrum`: the first in each ADT.
+            // In the future we will attempt to infer some or all of these from the catalog info.
+            SourceProfile.Point(
```

Comment:
Does this mean the user will see 05V and will have to select the correct one?

Reply:
Yup, they will see `Profile: Point`, `SED: Stellar Library` / `O5V`.

Reply:
Wouldn't it make more sense that it is empty?

Reply:
In the current model, there's no way to set it as empty.

The profile determines the band brightness units. So, in order to set the band brightness units from Simbad, we need to set a profile that accepts integrated units.

We could still make `SED` optional, but I think we decided on checking the user experience this way and then adjust the model if necessary.